event-processing/event-processing-template.yml

```diff
@@ -2708,6 +2725,10 @@ Resources:
       Runtime: nodejs14.x
       Timeout: 30
       Role: !GetAtt CleanserFunctionRole.Arn
+      Environment:
+        Variables:
+          SECRET_ARN: "{{resolve:ssm:HMACSecretArn}}"
```

Comment:
I might have misunderstood, but does the cleanser function actually need `HMACSecretArn`? Isn't it only using `PerformanceHMACSecretArn`?